deploy/helm/trickster/values.yaml

```diff
@@ -36,7 +54,9 @@ image:
 service:
   type: ClusterIP
   port: 9090
+  #address = "localhost"
```

Comment:
These (address/metricsAddress) aren't used to define the service, but for configuring the `listen` address for trickster. 

I'm not sure how relevant they are to the helm chart deployment -- do you see this changing at all? I don't see why it would be overridden.

I'm thinking you might want to:
- specify the ClusterIP here instead (that could get referenced in the service ~~and in the config instead of `address`~~)
- or, at deployment time you should get the pod IP in an env variable, if trickster could use that via the cli you could listen to a single IP
- or, not expose this? I'm not sure its useful if you can't specify the clusterIP.

Reply:
Thinking about this further -- I don't think you'd want to listen on the service address, but I could be mistaken. The pod IP could make sense.

Reply:
No, I think you are right. I don't think you would ever want to change the addr in the pod itself, just the clusterIP. Even at that I don't think I usually ever want to.